db/schema.rb

```diff
@@ -10,26 +10,34 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_03_10_001818) do
-
+ActiveRecord::Schema[7.0].define(version: 2023_01_04_193109) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "historical_matches", force: :cascade do |t|
     t.string "members", default: [], array: true
     t.string "grouping"
     t.date "matched_on"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
   end
 
   create_table "pending_notifications", force: :cascade do |t|
     t.string "strategy"
     t.date "last_attempted_on"
     t.bigint "historical_match_id"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
```

Comment:
Curious/non blocking Q: I feel like this is an OK change, and encountered similar on my WIP branch (#58) - is there a reason this was set previously or carryover of local a `db:migrate`?

Reply:
I don't really remember sadly. It's likely I overlooked adding the change awhile back, but not really sure.